src/cldr/loader.ts

```diff
@@ -24,6 +24,16 @@ function generateCldr(cldrData: any, locales: string[], includeInverse = false)
 	);
 }
 
+function getValidCldrDataLocale(locale: string): string {
+	const cldr = new Cldr(locale);
+	try {
+		require('cldr-data/main/${locale}/ca-gregorian.json');
```

Comment:
Try to require cldr-data for the locale, if unsuccessful use the min lang id from Cldr.